src/components/organisms/o-payment-gateway.vue

```diff
@@ -120,77 +96,58 @@ export default {
       BillText: "Bill description: ",
       bill: "Total : 2300 , discount : -300, GST : 500 , Total Amount: 2500 ",
```

Comment:
are we over-riding bill or its static as of now

Reply:
made it dynamic now